opencti-platform/opencti-graphql/src/database/file-storage.js

```diff
@@ -407,7 +407,7 @@ export const upload = async (context, user, filePath, fileUpload, opts) => {
 
   // Upload the data
   const readStream = createReadStream();
-  const fileMime = guessMimeType(key);
+  const fileMime = mime_type ?? guessMimeType(key);
```

Comment:
I did not change the behavior of guessMimeType ; it still returns plain/text by defaut.
Do we want to get rid of this too ?
What happens if there is no MIME type given and guess is impossible (maybe that's a dumb too theoretical question)

Reply:
I think we can still try to guess, but the fallback should be application/octet-stream rather than text/plain, what do you think?

Reply:
That would make more sens as a plain text generally have a file extension

Reply:
ok !